scripts/tool_installation.sh

```diff
@@ -56,6 +56,7 @@ After=network-online.target
 
 [Service]
 User=$USER
+EnvironmentFile=/etc/systemd/system/solana.env
```

Comment:
@NagaTulasi, where are the commands to create .env and store value from script.

Reply:
Just add the commands in script, which will be usable to read exported binary path just like config values and, we have to create .env files from script and include it in system service file.